data/hai/hai reveal 1 intro.txt

```diff
@@ -11,47 +11,436 @@
+	to offer
+		has "event: hr: pirates respond to leaks"
+		has "First Contact: Unfettered: offered"
+		not "Hai Reveal Concern Period: Raid on Allhome (local): offered"
+		random < 20
+	on offer
+		set "hr: allhome raided"
+		conversation
+			`A wall screen situated between two food stands just outside the docking area is playing through a news segment, and your ears suddenly perk up at the word "marauder".`
+			`	"Yesterday, a marauder terrorized the inhabitants of Allhome. They arrived by posing as a trader to investigate rumors currently circulating in human space about our existence. Hai security failed to identify the danger, and the individual snuck his ship in and landed at the spaceport without any opposition. Then his crew hijacked a vessel with some of the crew still on board and made off with it successfully. At this time it is unclear what has happened to those crewhai."`
+			`	At a table nearby, a Hai says, "Oh, that's awful!" and the human next to her responds, "Feels like being back on Solace."`
+			choice
+				`	(Approach the pair.)`
+				`	(Keep walking.)`
+					goto walking
+			
+			`	As you approach the human, a relatively young-looking man, glances up at you. "Oh hello, just arrived did you? You didn't happen to see an Argosy out there on your way in?"`
+			`	"Oh shush," says the Hai, "Your uncle will be here when said he will, no need to bother random folks just going about their business."`
+			`	She looks at you with a wary eye, but then seems to relent. "Though I suppose you did come over here of your own volition didn't you. Can we help you at all Captain?"`
+			choice
+				`	"Is this a common problem for Allhome?"`
+				`	"I heard you say this happened on Solace?"`
+					goto solace
+					
+			`	"Oh imperishables, no, not at all. It's completely unprecedented for something like that to happen, and in broad daylight too from the looks of the footage," she says.`
+			`	"It's like I was saying before, Auntie, ever since the retraction and that advisory, I swear no one except pirates is trying to get through the wormhole anymore."`
+			`	He looks at you seriously, "It's getting so bad that it's starting to feel like it did during the war, when all the militia were away and before the Free Worlds got a handle on the pirates. It seemed like every new ship that arrived was there to try and rob us, and it's the same now. We came here to be safe, but now it feels like all the government has managed to do is make sure that no one friendly is willing to risk the journey..."`
+				goto resolve
+			
+			label solace
+			`	The guy sits up a little straighter. "When the war started and most of the militia went away to the front, before the Free Worlds got a handle on the pirates, Solace essentially came under siege. Other than the regular merchants that we were familiar with, it seemed like every new ship that arrived was there to try and rob us. Either raiding the regulars, or raiding the planet.`
```

Comment:
```suggestion
			`	The young man sits up a little straighter. "When the war started and most of the militia went away to the front, before the Free Worlds got a handle on the pirates, Solace essentially came under siege. Other than the regular merchants that we were familiar with, it seemed like every new ship that arrived was there to try and rob us. Either raiding the regulars, or raiding the planet.`
```
'guy' sounds too casual for this serious conversation.

Also, I'd love to write a small mission about this during the FW campaign. Some sort of side mission.